form/form_customization.rst

```diff
@@ -337,7 +337,7 @@ You can also apply a form theme to a specific child of your form:
 
 .. code-block:: html+twig
 
-    {% form_theme form.child 'form/fields.html.twig' %}
+    {% form_theme form.children 'form/fields.html.twig' %}
```

Comment:
Well, the previous example was right in that it assumed that you would replace `child` with the name of the actual child form. Can we make that more clear with another example?

Reply:
see #9635